pkg/exporter/sumologicexporter/fields.go

```diff
@@ -24,15 +24,21 @@ import (
 
 // fields represents metadata
 type fields struct {
-	orig pdata.AttributeMap
+	orig        pdata.AttributeMap
+	initialized bool
```

Comment:
isn't there a way to figure it out using orig?

Reply:
Couldn't find a way. You can't access `pdata.AttributeMap` if it's uninitialized because it will cause a panic. We'd have to restructure code around `fields` a bit more to achieve this I guess.